ports/libmlservo/src/lib.rs

```diff
@@ -101,6 +84,16 @@ pub struct MLApp(*mut c_void);
 
 const LOG_LEVEL: log::LevelFilter = log::LevelFilter::Info;
 
+fn call<F>(f: F) -> Result<(), &'static str>
+where
+    F: FnOnce(&mut ServoGlue) -> Result<(), &'static str>,
+{
+    SERVO.with(|s| match s.borrow_mut().as_mut() {
```

Comment:
Sigh, `libsimpleservo` uses a global variable? This is a bit of a step back, `libmlservo` passed the servo instance around as a parameter.

Reply:
I'll file a followup issue to discuss this choice.